### ISSUE TITLE
Mongo

### ISSUE THREAD

**Issue by KamilNadlonek-TomTom** [Flaxoos/extra-ktor-plugins#54]
Hi
I am using `MongoDBLockManager` and I have problem when I want add second task with the same concurrency. 
Could you explain why you do not allow the creation of two tasks with the same concurrency but with different names?

When I debug code I found error message:
```
Write operation error on server localhost:27017. Write error: WriteError{code=11000, message='E11000 duplicate key error collection: archiver.TASK_LOCKS index: name_text_concurrencyIndex_1 dup key: { _fts: "archiv", _ftsx: 0.625, concurrencyIndex: 1 }', details={}}.
```

**Comment by Flaxoos** [comment]
Hi Kamil, sorry for the late response. let me look into it and get back to you

**Comment by Flaxoos** [comment]
@KamilNadlonek-TomTom This should be fixed in the new v2.0.0 version just released, can you please verify?

**Comment by Flaxoos** [comment]
Closing as fixed
